app/views/common/500.html.erb

```diff
@@ -6,7 +6,10 @@
 <p>
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %> <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
   <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%> <%= _("and relevant log entries.") %>
+  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%> <%= _("and relevant log entries, it is highly recommended to also attach the foreman-debug output.") %>
+</p>
+<p>
+  <%=  _("For more information on foreman-debug")%>  <%= documentation_button("7.3GettingHelp")%>
```

Comment:
The docs button looks rather big and a little lost here, would it look any better by removing this line of text and putting the Documentation button floating on the right of the above `<p>` section?